.gitignore

```diff
@@ -1,3 +1,5 @@
 _USER_DEFINES.h
 *backup*
 *aljaz*
+\.vscode
```

Comment:
These rules should go in your private global gitigore file.

Reply:
Yeah, I know, this is a legacy when porting the project from Arduino to PIO. Will fix it.